docs/configuration.rst

```diff
@@ -124,22 +181,25 @@ These settings control the behavior of django-csp. Defaults are in
 *italics*.
 
 ``CSP_REPORT_ONLY``
-    Send "report-only" headers instead of real headers. See the spec_
-    and the chapter on :ref:`reports <reports-chapter>` for more info. A
-    boolean. *False*
+    Send "report-only" headers instead of real headers.
+    A ``boolean``. *False*
+
+    See the spec_ and the chapter on :ref:`reports <reports-chapter>` for
+    more info.
+
 ``CSP_EXCLUDE_URL_PREFIXES``
-    A **tuple** of URL prefixes. URLs beginning with any of these will
+    A ``tuple`` of URL prefixes. URLs beginning with any of these will
```

Comment:
Add (not a ``list``) here to put emphasis on the fact that it may not be a list (in contrary to other settings)